tests/integration/index.ts

```diff
@@ -335,6 +335,47 @@ describe("Positive Cases", function () {
       expect(Number(liquidity)).to.be.closeTo(preComputeLiquidity.toNumber(), Number(convertToUnit(1, 14)));
       expect(shortfall).to.equal(0);
     });
+
+    it("Should revert when borrow cap is reached", async () => {
+      // Minting amount to account 1 so that it can also heal account
+      const fauceAmount = convertToUnit(20, 18);
```

Comment:
```suggestion
      const faucetAmount = convertToUnit(20, 18);
```

Reply:
Fixed in [8534bc9](https://github.com/VenusProtocol/isolated-pools/pull/286/commits/8534bc90af07ac0ae7d2081874ac57669bf7b7e5)